src/template/hook-template.sh

```
@@ -0,0 +1 @@
+# Run raphook
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)